vllm/spec_decode/spec_decode_worker.py

```diff
@@ -158,8 +170,9 @@ def create_worker(
                 proposer_worker = MedusaWorker(**draft_worker_kwargs)
             else:
                 if draft_tp == 1:
-                    draft_worker_kwargs[
-                        "model_runner_cls"] = TP1DraftModelRunner
+                    if current_platform.is_cuda_alike():
+                        draft_worker_kwargs[
+                            "model_runner_cls"] = TP1DraftModelRunner
```

Comment:
So in non-CUDA platforms this key will be missing. Is this expected?

Reply:
@comaniac , yes, `TP1DraftModelRunner` is a wrapper on ModelRunner, this wrapper will add extra data processing for 
 previous_hidden_states and other inputs before calling actual `model_runner.execute_model()`

In CPU enabling, we don't need extra work on previous_hidden_states to make it work, so `CPUTP1DraftModelRunner` is not necessary. However, in order to improve Spec Decode performance on other HWs, adding `HWTP1DraftModelRunner` might still be useful.